src/Bridge/Symfony/Routing/ApiLoader.php

```diff
@@ -146,6 +150,14 @@ private function loadExternalFiles(RouteCollection $routeCollection)
     {
         $routeCollection->addCollection($this->fileLoader->load('api.xml'));
 
+        if ($this->entrypointEnabled) {
+            $routeCollection->addCollection($this->fileLoader->load('api.xml'));
+        }
```

Comment:
doesn't this add 2 times the `api.xml` file ? :o

Reply:
(looking at the line above)

Reply:
thanks